msb/serial/config.py

```diff
@@ -0,0 +1,10 @@
+from msb.config.MSBConfig import MSBConf
+
+
+class SerialConf(MSBConf):
+    regex: str = "(\\d*\\.?\\d+)kW\\s*(\\d*\\.?\\d+)rpm\\s*(\\d*\\.?\\d+)rpm\\s*(\\d*\\.?\\d+)m/s\\s*(-?\\d*\\.*\\d*)"
```

Comment:
Regex could probably be simplified using `r""` string. The question is how the regex has to be defined in the YAML config. 🤔